src/pipeline/pipeline-context.js

```diff
@@ -1,6 +1,7 @@
 import * as urlUtils from '../utils/url';
 import * as contentUtils from '../utils/content';
 import { XHR_REQUEST_MARKER_HEADER } from '../const';
+import ContentCharset from '../utils/content-charset';
```

Comment:
Since `ContentCharset` now is not just a bunch of utility methods it has nothing to do with the utils and should be moved to `pipeline` and renamed to `Charset` I guess.